core/src/event.rs

```diff
@@ -2,25 +2,18 @@
 //! This module contains descriptions of such an events and
 //! utilitary Iroha Special Instructions to work with them.
 
-use std::{fmt::Debug, time::Duration};
+use eyre::{eyre, Result};
+use iroha_data_model::events::prelude::*;
```

Comment:
Minor style point: maybe move it to a separate block or into the same block as `crate::stream`?

Reply:
I'm not sure I get this comment

Reply:
We have blocks of imports: 
```
use iroha_imports 
...
<Blank-line>
use 3rd-party
...
<Blank-line>
use std::...
```
So I was saying that eyre is a 3rd-party import, and stylistically belongs in the 3rd-party block.

Reply:
I apologise, I hadn't realised that `cargo fmt` had a mind of its own on this subject.